jormungandr/cpp/autodiff/BindVariableMatrices.cpp

```diff
@@ -193,8 +205,17 @@ void BindVariableMatrix(py::module_& autodiff,
           blockCols = 1;
         }
 
-        return self.Block(rowOffset, colOffset, blockRows, blockCols);
+        return py::cast(self.Block(rowOffset, colOffset, blockRows, blockCols));
       });
+
+  variable_matrix.def("get_element",
+                      [](VariableMatrix& self, int row, int col) -> Variable& {
+                        if (row >= self.Rows() || col >= self.Cols()) {
+                          throw std::out_of_range("Index out of bounds");
+                        }
+                        return self(row, col);
+                      });
```

Comment:
This seems redundant and can be removed.

Reply:
Old test, forgot to clear. Removed now.